src/components/file-dropper/file-list-item.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from 'styled-components';
+import { theme } from '../layout/theme';
+import { TextStyles } from '../typography/typography-v2';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+
+interface FileListItemProps {
+  fileName: string;
+  fileCategories?: string[];
+  index: number;
+  onRemove: (index: number) => any;
+  onFileCategorySelect: (index: number, category: string) => any;
+}
+
+const FileListItemContainer = styled.div`
+  margin-top: 1px;
+  padding: 12px 16px;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  justify-content: space-between;
+  align-items: center;
+
+  background-color: ${theme.palette.background.leadbox};
+`;
+
+const FileName = styled.p`
+  ${TextStyles.headlineXS};
+  margin: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-wrap: none;
+`;
+
+const Actions = styled.div`
+  display: flex;
+  margin-left: auto;
+  padding-top: 1px;
+`;
+
+const FileCategoryDropdown = styled.select`
+  ${TextStyles.textR}
+  color: ${theme.palette.text.topline};
+
+  appearance: none;
+  border: none;
+  background: none;
+  cursor: pointer;
+  margin-right: 24px;
+
+  border-bottom: rgba(0, 0, 0, 0) 1px solid;
+  &:hover {
+    border-bottom: ${theme.palette.text.topline} 1px solid;
+  }
+`;
+
+const RemoveButton = styled.button`
+  ${TextStyles.textR}
```

Comment:
Not sure if the style is working if you don't end the line with `;`.

Reply:
1. Missing semicolons are silently ignored and causes the line being merged with the following statement. We need a linter to prevent that in general for styled components. 
2. In this case it's an interpolation and should resolve to the correct values stored in `textR` and a semicolon would be wrong I think, but I have never though about that. Worth to check the final part in css here. Maybe felix can post some observations>

Reply:
This seems to be a global problem, as it affects all new components that use a `<button />` or a `<select />`.

Reply:
I created an issue for it: https://github.com/satellytes/satellytes.com/issues/235